LocalWiki.php

```diff
@@ -178,13 +178,22 @@ function onInactiveSiteNoticeAfter( &$siteNotice, $skin ) {
 // Per-wiki settings
 if ( $wgDBname === 'erislywiki' ) {
 	$wgHooks['OutputPageParserOutput'][] = 'onOutputPageParserOutput';
```

Comment:
Will cause errors if this hook is not also changed with removal of param in function